README.md

```diff
@@ -0,0 +1,11 @@
+# Crypto-classical
+
+An educational tool for studying classical cryptography schemes. Do not encrypt anything of worth with this library.
+
+Included Ciphers:
+ - Caesar
+ -  Affine (Linear)
+ - Substitution
+ -  Stream
```

Comment:
A few of these aren't aligned properly.

Reply:
Don't know how this happened but I aligned them.